### DISCUSSION TITLE
My test of wall display

### DISCUSSION THREAD

**Discussion by ElMyggo** [UtilitechAS/amsreader-firmware#405] [Show and tell]
I test a setup to display all info on a 10 inch screen planned to mount on the wall..
I have a test module Wemos D1 mini with beta firmware (waiting for POW-K)
For display i use a cheap 10 inch monitor with HDMI
For computer i setup a Raspberry Pi Zero W as a kiosk viewer from this guide:
https://www.alessandroscola.com/en/computers/build-a-kiosk-with-raspberry
This is probably not a good solution 24/7 but i have to try ;-)

![kiosk](https://user-images.githubusercontent.com/111691693/213542778-0dc3adc1-d06b-4a43-ac3e-e7f5297f8924.jpg)

**Comment by ArnieO** [comment]
Wow - looks great! 👏

**Comment by ElMyggo** [comment]
Now with live data and integrated power for the Raspberry and power on/off button for safe shutdown

![screen1](https://user-images.githubusercontent.com/111691693/215189804-7d73dacb-8164-44e1-89bc-c20506c48838.jpg)
![screen2](https://user-images.githubusercontent.com/111691693/215189819-0d12e4c8-38c0-4c90-8da5-4528972f7581.jpg)
